doc/lua_api.txt

```diff
@@ -4555,7 +4555,8 @@ These functions return the leftover itemstack.
     * `param2` overrides `facedir` and wallmounted `param2`
     * `prevent_after_place`: if set to `true`, `after_place_node` is not called
       for the newly placed node to prevent a callback and placement loop
-    * returns `itemstack, success`
+    * returns `itemstack, success[, position]`
+      * `position`: the location the node was placed to. Nil if nothing was placed.
```

Comment:
```suggestion
      * `position`: the location the node was placed to. `nil` if nothing was placed.
```

And I suggest to maybe remove the `[]`s around `, position`, return values are not optional.